src/Conf.ml

```diff
@@ -1509,13 +1519,21 @@ let parse_line config ~from s =
               , Format.sprintf "expecting %s but got %s" Version.version
                   value ))
     | name, `File x ->
-        C.update ~config
-          ~from:(`Parsed (`File x))
-          ~name ~value ~inline:false
+        if !disable_conf_files then
+          Error (`Malformed "Disabled file configuration of ocamlformat")
```

Comment:
This will trigger an error when there is a `.ocamlformat` file in the current directory.

Reply:
Also the error message is pretty ugly:

```
Fatal error: exception ("User Error: malformed .ocamlformat file"
  ("invalid format" "Disabled file configuration of ocamlformat")
  ("invalid format" "Disabled file configuration of ocamlformat")
  ("invalid format" "Disabled file configuration of ocamlformat"))
```
(followed by stack trace)

Reply:
fixed